source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Digital wallets
+weight: 136
+---
+
+# Digital wallets
+
+You can enable [Google Pay](https://pay.google.com) and [Apple Pay](https://www.apple.com/uk/apple-pay/) [external links] to take payments from your users.
+
+You can only enable Google Pay and Apple Pay if you use [Worldpay](/switching_to_live/set_up_a_live_worldpay_account/#set-up-a-live-worldpay-account) as your payment service provider (PSP).
```

Comment:
This will not work in sandbox nor should you be able to enable these in sandbox. So I think it'd be better to say you must have a *live WorldPay account* or words to that effect.

Reply:
agree